cmd/nitro/nitro.go

```diff
@@ -255,8 +255,11 @@ func mainImpl() int {
 
 	nodeConfig.Node.EspressoCaffNode.ResolveDirectoryNames(nodeConfig.Persistent.Chain)
 
-	if nodeConfig.Node.EspressoCaffNode.Enable && nodeConfig.Node.EspressoCaffNode.EspressoTeeType != "" {
-		teeAddress, err = integrityattestation.ReadEnclaveAddress(nodeConfig.Node.EspressoCaffNode.KeyPairAttestationsPath)
+	var caffNodetxOpts *bind.TransactOpts
+
+	if nodeConfig.Node.EspressoCaffNode.Enable {
+		var key *ecdsa.PrivateKey
+		key, err = integrityattestation.ReadEnclavePrivateKey(nodeConfig.Node.EspressoCaffNode.KeyPairAttestationsPath)
```

Comment:
I think you also need to check if `nodeConfig.Node.EspressoCaffNode.EspressoTeeType != "" ` because if caff node is not running in TEE we dont need this